tests/functional/services/flows/test_list_runs.py

```diff
@@ -68,3 +69,37 @@ def test_list_runs_filter_flow_id(flows_client, pass_as_uuids):
     )
     for run in res_combined:
         assert run["flow_id"] in {str(flow_id_one), str(flow_id_two)}
+
+
+@pytest.mark.parametrize(
+    "filter_roles, expected_filter_roles",
+    [
+        # empty list
+        ([], []),
+        # list with empty string
+        ([""], [""]),
+        # None
+        (None, None),
```

Comment:
The way this was written actually taught me a bit of corner-case behavior in `requests` that I didn't know about. I didn't realize that passing an empty string resulted in a parameter being _omitted_. (Remember that we're going through `utils.commajoin`)
However, I think the way the test is written gives a different impression about what's being asserted.

I tested this rewrite locally and it does the same thing (and passes):
```suggestion
        # empty list, list with empty string, and None do not send the param
        ([], None),
        ([""], None),
        (None, None),
```

I think this is much better at communicating what the test asserts -- the variable right-hand-side values in these parameters makes it look like we're testing three different outcomes. Making them all `None` clarifies that we're testing for the same outcome.